Expensify/App
# [No QA] Setup Flipper plugins when on native & dev

### Details

This PR adds some Flipper plugins to dev. These are things I have used and have found useful. I'm not creating an issue as these plugins are opt-in developer efficiency tools. If you like them use them. If not, ignore them. If there are any questions about how to use them I will answer them and we can create SO posts if needed. It's all pretty self explanatory once you install Flipper and build the app.

I created an SO post here to describe how to setup Flipper -> https://stackoverflow.com/c/expensify/questions/10490/10491

### Fixed Issues
$ No Issue

### Tests
1. Make sure the web side is unaffected by these changes
2. Install Flipper -> https://fbflipper.com/
3. Build the development version of the app
4. Verify you are able to Flipper

### QA Steps


### Tested On


### Screenshots

## React DevTools
![2021-07-28_14-27-44](https://user-images.githubusercontent.com/32969087/127413103-303df23d-ecc8-4474-8106-ea7cc39b4572.png)
![2021-07-28_14-27-40](https://user-images.githubusercontent.com/32969087/127413108-7596b2f1-2310-41f8-a9cf-9ca4e95e97a4.png)

## AsyncStorage
![2021-07-28_14-26-55](https://user-images.githubusercontent.com/32969087/127413118-75b73c20-1d59-40e9-93eb-1f2ee989ae05.png)

## Native layout
![2021-07-28_14-26-24](https://user-images.githubusercontent.com/32969087/127413133-a13174d6-4d47-43f2-8ed0-c1c5195b69ac.png)

## React Native bridge
![2021-07-28_14-25-58](https://user-images.githubusercontent.com/32969087/127413150-6ad1351c-59df-4638-af26-9bb2e798b727.png)

## Network traffic
![2021-07-28_14-25-44](https://user-images.githubusercontent.com/32969087/127413158-8d8fd2ab-f512-48f8-892f-3b738350f982.png)

## JS Console
![2021-07-28_14-23-42](https://user-images.githubusercontent.com/32969087/127413168-3777d42f-106b-4706-9f3d-258fe329a3d4.png)

## Device Logs
![2021-07-28_14-23-23](https://user-images.githubusercontent.com/32969087/127413172-23b48893-491d-476f-a90b-8b05089fea28.png)
